pkg/catalog/diff.go

```diff
@@ -7,6 +7,7 @@ const (
 	DifferenceTypeRemoved
 	DifferenceTypeChanged
 	DifferenceTypeConflict
+	DifferenceTypeCommonPrefix
```

Comment:
We should probable be able to count these separately, unlike what we do above.  Not having to add a stringification for the new type is something of a smell.

Reply:
Good point.

Reply:
Removed it.